File: Sources/SimERP/ClientApp/src/app/lists/customer/model/customercommission.ts
export class CustomerCommission {
    RowId: number;
    BeneficiaryName: string = "";
    PhoneNumber: string;
    Email: string;
    BankAccount: string;
    BankName: string;
    Notes: string;
    CreatedBy: number;
    CreatedDate?: Date;
    ModifyBy?: number;
    DateTimeOffset?: Date;
    IsActive: boolean = true;
    CustomerId: number;
}

File: Sources/SimERP.Data/DBEntities/CustomerCommission.cs
using System;
using System.Collections.Generic;

namespace SimERP.Data.DBEntities
{
    public partial class CustomerCommission
    {
        public long RowId { get; set; }
        public int CustomerId { get; set; }
        public string BeneficiaryName { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string BankAccount { get; set; }
        public string BankName { get; set; }
        public string Notes { get; set; }
        public int? CreatedBy { get; set; }
        public DateTimeOffset? CreatedDate { get; set; }
        public int? ModifyBy { get; set; }
        public DateTimeOffset? ModifyDate { get; set; }
        public bool IsActive { get; set; }
    }
}
